components/blocks/verticalListItem.tsx

```diff
@@ -12,8 +13,40 @@ export const VerticalListItem = ({ data }) => {
           <h3 className="text-sswRed">{data.header}</h3>
         </div>
       )}
-      <div className={classNames("flex flex-row items-center")}>
-        {data.icon && (
+      <div className={classNames("flex flex-row items-center font-helvetica")}>
+        {data.icon && isNumber(data.icon) ? (
+          <svg
+            width="45"
+            height="45"
+            viewBox="0 0 42 42"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            className="mr-5 text-sswRed"
+          >
+            <g>
+              <circle
+                cx="21"
+                cy="21"
+                r="21"
+                fill="currentColor"
+                fillOpacity="0.1"
+              />
+              <text
+                x="50%"
+                y="50%"
+                height="19"
+                fontSize="20"
+                fontWeight="700"
+                textAnchor="middle"
+                fill="currentColor"
+                fontFamily="helvetica"
+                dominantBaseline="central"
+              >
+                {data.icon}
+              </text>
+            </g>
+          </svg>
```

Comment:
Can we move the SVG to its own file?

Reply:
Save circle icon to `images/icons/circle-icon.svg`